modules/ServerAPI/src/server/routes/InventoryServiceRoutes.coffee

```diff
@@ -1367,22 +1367,14 @@ exports.moveToLocationInternal = (input, callCustom, updateLocationHistory, call
 				#add the call to updateContainerHistoryLogs here...
 				console.debug "response statusCode: #{response.statusCode}"
 				if !error
-					console.log input[0].containerCodeName
-					if updateLocationHistory
-						exports.updateContainerHistoryLogsInternal(input, (json, statusCode) ->
-							callback json, statusCode
-							if callCustom && csUtilities.moveToLocation?
-								console.log "running customer specific server function moveToLocation"
-								csUtilities.moveToLocation input, (customerResponse, statusCode) ->
-									json = _.extend json, customerResponse
-									callback json, statusCode
-							else
-								if callCustom
-									console.warn "could not find customer specific server function moveToLocation so not running it"
+					shouldCallCustom = (callCustom && csUtilities.moveToLocation?)
+					callFunctionOrReturnNull shouldCallCustom, csUtilities.moveToLocation, input, (customerResponse, statusCode) ->
+						if updateLocationHistory
+							exports.updateContainerHistoryLogsInternal(input, (json, statusCode) ->
 								callback json, statusCode
-						)
-					else
-						callback json, response.statusCode
+							)
+						else
+							callback json, response.statusCode
```

Comment:
I think if it gets to this else block you'll get an error right? json isn't defined.

Reply:
why would I get an error?why would I get an error?